catalog/transformer/catalog-transformer-pdf/src/test/groovy/ddf/catalog/transformer/input/pdf/PdfInputTransformerSpecTest.groovy

```diff
@@ -43,28 +53,23 @@ class PdfInputTransformerSpecTest extends Specification {
     }
 
     void setup() {
-        bundleMock = Mock(Bundle)
-
-        pdfInputTransformer = new PdfInputTransformer() {
-            @Override
-            def Bundle getBundle() {
-                return bundleMock
-            }
-        };
+        metacardType = new MetacardTypeImpl("pdf", [
+            new AssociationsAttributes(),
+            new ContactAttributes(),
+            new MediaAttributes(),
+            new DateTimeAttributes(),
+            new LocationAttributes(),
+            new ValidationAttributes()
+        ])
+
+        pdfInputTransformer = new PdfInputTransformer(metacardType, false);
     }
 
     void verifySamplePdfMetacard(Metacard metacard) {
         verifyThumbnailSize(metacard.thumbnail)
-        assert metacard.contentTypeName == 'pdf'
-        assert metacard.title == 'Microsoft Word - Document1'
+        assert metacard.contentTypeName == 'application/pdf'
         assert metacard.createdDate.getTime() == 1456150156000
```

Comment:
the test pdf doesn't have author or keywords, and i don't yet have a good way to edit the pdf metadata